### PR TITLE
Updating installation documentation

### PR DESCRIPTION
**Pull request by genevievestarke**
Updating the documentation page to reflect most current installation instructions

### CODE REVIEW

File: docs/install_instructions.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Still a work in progress but currently there are 3 sets of installation instructions
 
-[Installing on kestrel](install_on_kestrel)
+[Local installation instructions](intall_local)
```
Comment (jfrederik-nrel) [inline docs/install_instructions.md:5]: *install

**Review by jfrederik-nrel — APPROVED** [review]
Just the one typo

File: docs/install_local.md [hunk 2]
```diff
@@ -136,6 +134,20 @@ launches the hercules emulator. These will connect to the helics_broker and
...
+```
+./bash_script.sh
+```
+However, if the simulation hangs, be sure to check if there are multiple processes running with 
+```
+ps
+```
+which will tell you all of the processes currently working.  You can kill processes that you do not want by using the kill command, paired with the process number (filled into the blank in the command below).
+```
+kill ___
```
Comment (misi9170) [inline docs/install_local.md:148]: Change to `kill <PID>`?

**Review by misi9170 — APPROVED** [review]
Approving now; feel free to merge when you're ready

**Pull request merged**